files/sublime_settings/JsPrettier.sublime-settings

```diff
@@ -0,0 +1,3 @@
+{
+  "auto_format_on_save": true
```

Comment:
This file won’t copy over automatically, I think you’ll need to adjust the sublime_settings.sh file.